build_projects/dotnet-cli-build/GetNuGetInformation.cs

```diff
@@ -1,23 +1,22 @@
-// Copyright (c) .NET Foundation and contributors. All rights reserved.
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using Microsoft.Build.Framework;
 using Microsoft.Build.Utilities;
-
-using static Microsoft.DotNet.Cli.Build.Framework.BuildHelpers;
+using Microsoft.DotNet.Cli.Build;
 
 namespace Microsoft.DotNet.Cli.Build
 {
-    public class FixModeFlags : Task
+    public class GetNuGetInformation : Task
```

Comment:
Fair point. We've diverged enough from NuGet here that there is no sense planning for asking NuGet where the package cache lives. WIll fix.